test/strings/basic.jl

```diff
@@ -107,6 +107,10 @@ end
 @test SubString("", 1, 6)[10:9] == ""
 @test SubString("", 1, 0)[10:9] == ""
 
+# issue #22500 (using `get()` to safely index strings)
+@test get("Julia", 1, ' ') == 'J'
+@test get("Julia", -1, ' ') == ' '
+@test get("Julia", 10, ' ') == ' '
```

Comment:
Excellent.  This is exactly the kind of sideways thinking I want when I open a PR.  Addressed in my new squashed commit.